spring-boot-admin-server/pom.xml

```diff
@@ -78,6 +78,11 @@
             <artifactId>spring-boot-configuration-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <!-- Optional JSON serialisation for MS Teams Notifications  -->
+        <dependency>
+          <groupId>org.json</groupId>
```

Comment:
Please remove this additional dependency. Setting the content-type header for the request should do the job, as it's most likely that a json serializer is on the classpath.

Reply:
Ok - removed